cyences_app_for_splunk/README/savedsearches.conf.spec

```diff
@@ -0,0 +1,7 @@
+
+[default]
+display.visualizations.custom.radial_flow_map.radial_flow_map.defaultWeight = <number>
```

Comment:
Change name for `radial_flow_map` to `network_telemetry_map`.

Reply:
This is required for btool. Otherwise btool will throw warnings.